src/cmd-buildextend-live

```diff
@@ -415,6 +415,10 @@ def generate_iso():
         # can be EFI/fedora or EFI/redhat
         grubpath = glob.glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
         shutil.move(grubpath[0], os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
+        for key in list(files_with_karg_embed_areas):
+            if re.search(r"/EFI/.*/grub.cfg", key):
+                files_with_karg_embed_areas['/srv/fcos/tmp/buildpost-live/live/boot/grub/grub.cfg'] = files_with_karg_embed_areas[key]
```

Comment:
I think you can capture the `os.path.join()` output above where we create the `boot/grub/grub.cfg` path and reuse that here.

Reply:
Thanks Lebon I added that  "os.path.join(tmpisoroot, 'boot/grub/grub.cfg') "

Reply:
Still seems like we're doing `os.path.join()` twice.

Reply:
This doesn't look resolved.

Reply:
https://github.com/coreos/coreos-assembler/pull/2831/commits/4bc93db282e2e4c45faa8d6919ab2e26b10a0a90